app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TweetController < ApplicationController
+
+
+  get '/tweets' do
+    if logged_in?
+      @user = User.find_by(id: session[:user_id])
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = User.find_by(id: session[:user_id])
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      if params[:content] == " " #does not allow creation of new tweet if contents are blank
+        redirect to '/tweets/new'
+      else
+        @tweet = Tweet.create(content: params[:content], user_id: session[:user_id])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.